DCode.Common/EmailHelper.cs

```diff
@@ -53,6 +53,7 @@ public static void SendEmail(string toMailAddress, string ccMailAddress, MailMes
 
                     SmtpServer.Port = 25;
                     SmtpServer.Credentials = new System.Net.NetworkCredential(ConfigurationManager.AppSettings[Constants.DcodeEmailId], ConfigurationManager.AppSettings[Constants.DcodeEmailPwd]);
+                    SmtpServer.UseDefaultCredentials = true;

```

Comment:
Let's have this removed and leave the email sending mechanism as is.

Reply:
Sure. I had put that for testing purpose.